.circleci/config.yml

```diff
@@ -268,11 +263,22 @@ workflows:
           requires:
             - cargo_fetch
       - ffi_regression:
+          sector_size: 1024
           requires:
             - cargo_fetch
+            - obtain_parameters
+      - ffi_regression:
```

Comment:
would be good to name this sth like `e2e_256MiB` and `e2e_1KiB`, so the github checks are easier to understand